src/resources/lang/en/en.json

```diff
@@ -1322,6 +1326,8 @@
   "auto_bill_ach_date_help": "ACH will always auto bill on the due date.",
   "warn_change_auto_bill": "Due to NACHA rules, changes to this invoice may prevent ACH auto bill.",
   "bank_account": "Bank Account",
+  "connect_accounts": "Connect accounts",
+  "bank_account_name": "Bank Account Name",
```

Comment:
Never edit `en.json` file directly in the app. It is automatically generated from the backend. In case of missing translation, please contact me or David.

Reply:
> Never edit `en.json` file directly in the app. It is automatically generated from the backend. In case of missing translation, please contact me or David.

Removed